index.html

```diff
@@ -814,150 +814,161 @@ <h2>Privacy Considerations</h2>
-      "proofValue": "z5SpZtDGGz5a89PJbQT2sgbRUiyyAGhhgjcf86aJHfYcfvPjxn6vej5na6kUzmw1jMAR9PJU9mowshQFFdGmDN14D",
-      "proofPurpose": "assertionMethod",
-      "verificationMethod": "https://example.com/issuer/123#key-0"
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://www.w3.org/2018/credentials/examples/v1",
+    "https://w3id.org/security/suites/ed25519-2020/v1"
+  ],
+  "id": "http://example.edu/credentials/3732",
+  "type": [
+    "VerifiableCredential",
+    "UniversityDegreeCredential"
+  ],
+  "issuer": "https://example.edu/issuers/565049",
+  "issuanceDate": "2010-01-01T00:00:00Z",
+  "credentialSubject": {
+    "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+    "degree": {
+      "type": "BachelorDegree",
+      "name": "Bachelor of Science and Arts"
     }
   },
-  "vp_0": {
-    "@context": [
-      "https://www.w3.org/2018/credentials/v1",
-      "https://w3id.org/security/suites/ed25519-2020/v1"
-    ],
-    "type": ["VerifiablePresentation"],
-    "verifiableCredential": [
-      {
-        "@context": [
-          "https://www.w3.org/2018/credentials/v1",
-          "https://www.w3.org/2018/credentials/examples/v1",
-          "https://w3id.org/security/suites/ed25519-2020/v1"
-        ],
-        "id": "http://example.gov/credentials/3732",
-        "type": ["VerifiableCredential", "UniversityDegreeCredential"],
-        "issuer": "https://example.com/issuer/123",
-        "issuanceDate": "2020-03-10T04:24:12.164Z",
-        "credentialSubject": {
-          "id": "did:example:456",
-          "degree": {
-            "type": "BachelorDegree",
-            "name": "Bachelor of Science and Arts"
-          }
-        },
-        "proof": {
-          "type": "Ed25519Signature2020",
-          "created": "2019-12-11T03:50:55Z",
-          "proofValue": "z5SpZtDGGz5a89PJbQT2sgbRUiyyAGhhgjcf86aJHfYcfvPjxn6vej5na6kUzmw1jMAR9PJU9mowshQFFdGmDN14D",
-          "proofPurpose": "assertionMethod",
-          "verificationMethod": "https://example.com/issuer/123#key-0"
-        }
-      }
-    ],
-    "id": "urn:uuid:83895ddf-52ee-4408-8796-51a1856dbbec",
-    "holder": "did:ex:12345",
-    "proof": {
-      "type": "Ed25519Signature2020",
-      "created": "2021-06-04T20:50:09Z",
-      "verificationMethod": "https://example.com/issuer/123#key-0",
-      "proofPurpose": "authentication",
-      "challenge": "123",
-      "proofValue": "z2y3UBXAiToXLzQqeMnHiMozJ3hKxcMgLm7p8GRQA92F6JSYu49RxHQf6k1CMKnMdpj3BLRSH69b9qA9cfjE3oS7q"
-    }
+  "proof": {
+    "type": "Ed25519Signature2020",
+    "created": "2022-12-07T21:31:08Z",
+    "verificationMethod": "https://example.edu/issuers/565049#key-1",
+    "proofPurpose": "assertionMethod",
+    "proofValue": "z2RczMj342tVhAjgjEPV4TeHbi2ggnTRKTc5BFQCgaWJ3nhcg5HgCeC2eV4Lc1fYdhfoLyPjxoq4BtqrsyNvxZ8nE"
   }
 }
       </pre>
+
+    </section>
+      <section>
+        <h3>Representation: DataIntegrityProof</h3>
+        <p>To be written...</p>
```

Comment:
maybe add a link to an issue.

Reply:
Agree with @OR13, this feels out of place. Either link to an issue, or remove it from the PR (since, presumably, this section will appear in a future PR).

Reply:
Agree. Will update and reissue.